### ISSUE TITLE
Organization Address cutting off important detail

### ISSUE THREAD

**Issue by Staxed** [SparkDevNetwork/Rock#2000]
### PLEASE READ

If the nature of your issue relates to the security of Rock, please do not post it here in the public issues.

### Prerequisites

* [x] Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
* [x] Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

[When setting Organization Address Value, it seems to cut off after a certain point.  Our mailing address is at a local mail collection place, so we have the street name along with the unit and box number (111 1st St STE 1 #111 )]

### Steps to Reproduce

1. [Admin > General Settings > Global Attributes > Organization Address ]
2. [Input "111 1st St STE 1 #111" and save]

**Expected behavior:** 

[Address is updated exactly as typed]

**Actual behavior:** 

[Truncates to 111 1st St, and removes STE 1 #111]

### Versions

* **Rock Version:** [6.1]  
* **Client Culture Setting** [Enter Client Culture here]

**Comment by gladsonj** [comment]
@Staxed Is this still an issue? Seems to be working fine in the Demo Site.

![image](https://cloud.githubusercontent.com/assets/21976051/24299161/9ee3efa2-107e-11e7-8cc3-0cdde1fabfca.png)

**Comment by Staxed** [comment]
@gladsonj Using the example above it does seem to work, that was just made up info in the same format as the actual input I need.  I did just test it out again and it doesn't work with our actual address "1605 Central Ave STE 6 # 233", results in 1605 Central Ave.

Though, I just tried it on the demo site and it works fine, but still not working on our own install (though it's all stock settings and files).
